svn/trunk/inc/class-hyvor-talk-webpage.php

```diff
@@ -130,11 +130,14 @@ public function shortCodeComments($attr) {
 
 	public function shortCodeCount($attr) {
 		$this->setEmbedVariables(false);
+		ob_start();
 		include_once HYVOR_TALK_DIR_PATH . '/html/count.php';
+		$content = ob_get_clean();
 
-		$mode = isset($attr['mode']) && $attr['mode'] === 'number' ? "data-talk-mode=\"number\"" : '';
+		$pageId = isset($attr['id']) ? $attr['id'] : '';
+		$mode = isset($attr['mode']) && $attr['mode'] === 'number' ? "mode=\"number\"" : '';
 
-		$content = "<span data-talk-id=\"{$attr['id']}\" $mode></span>";
+		$content .= "<hyvor-talk-comment-count page-id={$pageId} {$mode}></hyvor-talk-comment-count>";
```

Comment:
I think you'll need quotes around `page-id="{$pageId}"`, otherwise it won't work with some IDs

Reply:
if we put quotes around `pageId`, in case of a str it would like `""pageID""`, isn't it?